### ISSUE TITLE
RetinaFace on Keras

### ISSUE THREAD

**Issue by edumucelli** [deepinsight/insightface#862]
That is not an issue, but a question. Have someone converted RetinaFace to Keras or Tensorflow model?

**Comment by sonfiree** [comment]
Any success?

**Comment by StanislasBertrand** [comment]
Hello, I just finished converting retinaface to tf2+, code is in this repo
https://github.com/StanislasBertrand/RetinaFace-tf2

Accuracy is as good, but inference is a little bit slower

Any feedback appreciated !
